include/bluetooth/hci.h

```diff
@@ -1856,6 +1856,48 @@ typedef bool bt_hci_vnd_evt_cb_t(struct net_buf_simple *buf);
   */
 int bt_hci_register_vnd_evt_cb(bt_hci_vnd_evt_cb_t cb);
 
+
+/** Allocate an HCI event buffer.
+ *
+ * This function allocates a new buffer for an HCI event. It is given the
+ * avent code and the total length of the parameters. Upon successful return
+ * the buffer is ready to have the parameters encoded into it.
+ *
+ * @param evt        Event OpCode.
+ * @param len        Length of event parameters.
+ *
+ * @return Newly allocated buffer.
+ */
+struct net_buf *bt_hci_evt_create(u8_t evt, u8_t len);
```

Comment:
Are we sure `include/bluetooth/hci.h` is the most appropriate place for these? Wouldn't it always be the HCI driver that's the user of these, which would then make `include/drivers/bluetooth/hci_driver.h` seem like the more appropriate place?

Reply:
Sure, I will move them there.